platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -123,10 +139,11 @@ include the following:
    spaces and supporting services to deploy and use those third-party products and 
    services
 
-A more advanced platform also _composes_ these capabilities into experiences and 
-templates fit for key scenarios like web application development or data analysis. 
-For example, an application developer platform can include templates for provisioning 
-complete development environments and could serve the following use cases:
+A more advanced internal platform also _composes_ capabilities into experiences
+and templates fit for key scenarios like web application development or data
+analysis. For example, an internal developer platform includes templates for
+provisioning complete development environments and could serve the following use
+cases:
```

Comment:
I just clocked that we talk about templates here and not on-demand services. IMO if we are describing "mature" here we need to step away from templates (which still leave ownership with the consuming team to use them) and all the way to on demand.

```suggestion
analysis. For example, an internal developer platform provide on demand 
provisioning of complete development environments and could serve the following use
cases:
```

Reply:
I hear you and will try to add self-service and other short adjectives here. I think I'm assuming that templates, like APIs, depend on a self-serviceable implementation that reifies whatever they declare, so perhaps I can express that.

Reply:
In my last commit I combined the two lists of use cases and mentioned self-service and templates, PTAL!